repose-aggregator/components/services/phone-home-service/src/main/scala/org/openrepose/core/services/phonehome/PhoneHomeService.scala

```diff
@@ -186,7 +188,9 @@ class PhoneHomeService @Inject()(@Value(ReposeSpringProperties.CORE.REPOSE_VERSI
           ))
         }
       } catch {
-        case e: Exception => logger.error("Could not send an update to the collection service", e)
+        case e: Exception =>
+          logger.error("Could not send an update to the collection service", e)
+          scope.span.setTag(Tags.ERROR.getKey, true)
```

Comment:
I could replace this with:
```
tracer.activeSpan.setTag(...)
```

That would allow us to remove the scope parameter to this method. I kind of liked being explicit, but I would be happy to change this if someone prefers the other way.

Reply:
I like the explicitness of passing the `Scope` in too.